package.json

```diff
@@ -1,18 +1,20 @@
 {
     "workspaces": [
         "packages/app",
-        "packages/tests"
+        "packages/tests",
+        "packages/client"
     ],
     "private": true,
     "scripts": {
         "program:lint": "cargo fmt --all -- --check && cargo clippy -- --deny warnings",
         "program:dylint": "cargo dylint --all --workspace",
-        "lint": "eslint .",
+        "lint": "eslint . --fix --ext .ts",
```

Comment:
I prefer to separate lint and auto-format, but not strongly opinionated

Reply:
Yeah, this needs to be reverted or CI will try to fix them.

Best to just write `yarn lint --fix` locally. If you like we can add a `yarn lint:fix`

Also if there is consensus we can add husky

Reply:
> Yeah, this needs to be reverted or CI will try to fix them.
> 
> Best to just write `yarn lint --fix` locally. If you like we can add a `yarn lint:fix`
> 
> Also if there is consensus we can add husky

100% agree and like

Reply:
Let's do it as a separate PR to avoid overloading this one